superset/migrations/versions/0a6f12f60c73_add_role_level_security.py

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_role_level_security
+
+Revision ID: 0a6f12f60c73
+Revises: 817e1c9b09d0
+Create Date: 2019-12-04 17:07:54.390805
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "0a6f12f60c73"
+down_revision = "817e1c9b09d0"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    op.create_table(
+        "row_level_security_filters",
+        sa.Column("created_on", sa.DateTime(), nullable=True),
```

Comment:
All these columns should be in the model.

Reply:
I think they are in the model via the `mixin`s.  Am I wrong about this?  These migrations were automatically created using `superset db migrate` as per the contributing readme

Reply:
Ah, forgot about that. Nvm.